File: apps/backend/scripts/fix_indent.py
lines = []
with open("apps/backend/api/v1/content.py", "r", encoding="utf-8") as f:
    lines = f.readlines()

# 1-indexed to 0-indexed: 414 -> 413, 802 -> 801
start_idx = 413
end_idx = 802

with open("apps/backend/api/v1/content.py", "w", encoding="utf-8") as f:
    for i, line in enumerate(lines):
        if start_idx <= i <= end_idx:
            # Add 4 spaces only if line is not empty (preserve empty lines logic if needed, but python tolerates indented empty lines)
            if line.strip():
                f.write("    " + line)
            else:
                f.write(line)
        else:
            f.write(line)

print("Indentation fixed.")

File: apps/backend/scripts/fix_indent.js
const fs = require('fs');
const path = require('path');

const filePath = path.join(__dirname, '..', 'api', 'v1', 'content.py');
console.log(`Processing ${filePath}`);

const lines = fs.readFileSync(filePath, 'utf8').split('\n');
const startIdx = 413; // 414th line (0-indexed)
const endIdx = 802;   // 803rd line (0-indexed)

const newLines = lines.map((line, idx) => {
    if (idx >= startIdx && idx <= endIdx) {
        if (line.trim().length > 0) {
            return '    ' + line;
        }
    }
    return line;
});

fs.writeFileSync(filePath, newLines.join('\n'));
console.log('Indentation fixed.');
